sql/core/src/main/java/org/apache/spark/sql/vectorized/ArrowColumnVector.java

```diff
@@ -455,9 +455,9 @@ final boolean isNullAt(int rowId) {
     @Override
     final ColumnarArray getArray(int rowId) {
       ArrowBuf offsets = accessor.getOffsetBuffer();
-      int index = rowId * accessor.OFFSET_WIDTH;
+      int index = rowId * BaseRepeatedValueVector.OFFSET_WIDTH;
```

Comment:
I had used `ListVector` here although it's actually defined in the superclass. `ListVector` was the type of `accessor`. Either is OK and I'd generally favor your change; `ListVector` is super conservative.